README.md

```diff
@@ -13,10 +13,6 @@ To make use of library you would need some interfaces and contracts to be [compi
 and injected by brownie into your brownie project.
 List of required interfaces can be found [here](https://github.com/SHAKOTN/badger-utils/tree/master/interfaces)
 
-You also need some contracts to be compiled as well:
-```
-ForceEther, SafeMath, Token, BadgerRegistry
-```
```

Comment:
looks to me like `ForceEther` is still required by `distribute_from_whales_realtime.py` L36?

Reply:
Ok, I will move this description back